### DISCUSSION TITLE
Ubuntu

### DISCUSSION THREAD

**Discussion by caillou100** [bol-edu/HLS-SOC-Discussions#196] [Q&A]
在這次實作的指令是否有辦法在windows上去做執行,因為如果本身電腦的記憶體不大,CPU不給力的話就會跑得很慢,想請各位大神是否有方法在windows上執行.

**Comment by kevin861222** [comment]
同學你好，我不確定你所言的「這次實作」是指哪一個 lab ，但你如果想要在 windows 環境上執行應該也是可行的，vivado 和 Prebuilt RISC-V GCC Toolchains 都有 windows 版本，不過實際運行起來穩不穩定我就沒辦法保證了
https://gnutoolchains.com/risc-v/
